src/components/Input.react.js

```diff
@@ -59,7 +61,7 @@ export default class Input extends Component {
   render() {
     var benefits = {taxes: 1000}; // This is a placeholder for the benefits that we'll know they get
     return(
-    <Grid>
```

Comment:
This was causing another container to appear within the main page container defined in app. The result was that everything in input was thrown off a bit.